src/main/kotlin/com/github/seliba/devcordbot/command/context/Context.kt

```diff
@@ -72,14 +72,14 @@ data class Context(
     /**
      * The member of the [author].
      */
-    val member: Member
-        get() = message.member!! //CommandClient ignores webhook messages, so this cannot be null
+    val member: Member?
+        get() = if (message.isFromType(ChannelType.TEXT)) message.member else bot.guild.getMemberById(author.id)
 
     /**
      * The guild of the [channel].
      */
     val guild: Guild
-        get() = message.guild
+        get() = if (message.channelType == ChannelType.TEXT) message.guild else bot.guild
```

Comment:
It's bad practice to store Discord entities for a long time (like the whole lifetime of the current session)
```It is not recommended to store any of these entities for a longer period of time! Instead of keeping (e.g.) a User instance in some field, an ID should be used. With the ID of a user, you can use getUserById(id) to get and keep the user reference in a local variable (see below).``` https://github.com/DV8FromTheWorld/JDA#entity-lifetimes

Reply:
I need the member for the Private message commands. The bot is bound to a guild, so that is valid?

Reply:
IDK what exactly can happen to the Guild entity representing DevCord but, just to be safe I would read it from cache each time, shouldn't be a huge performance impact

Reply:
That means `bot.guild` instead of `if (message.channelType == ChannelType.TEXT) message.guild else bot.guild`

Reply:
We should make bot.guild a getter for jda.getGuildById then it should be fine